vector/src/main/res/layout/adapter_item_vector_message_room_member.xml

```diff
@@ -77,12 +79,12 @@
                             android:layout_height="wrap_content"
                             android:layout_gravity="left"
                             android:layout_marginLeft="4dp"
+                            android:layout_toLeftOf="@+id/message_timestamp_layout"
                             android:layout_toRightOf="@id/message_adapter_e2e_icon"
                             android:autoLink="all"
                             android:text="A body"
```

Comment:
Why don't use tools:text = "A body"

Reply:
yes, a global replacement will be done later